File: SDKs/googleapis.com/metastore/v1beta/typescript/src/sdk/models/shared/consumer.ts
import { Metadata, SpeakeasyBase } from "../../../internal/utils/utils";


// Consumer
/** 
 * Contains information of the customer's network configurations.
**/
export class Consumer extends SpeakeasyBase {
  @Metadata({ data: "json, name=endpointUri" })
  endpointUri?: string;

  @Metadata({ data: "json, name=subnetwork" })
  subnetwork?: string;
}

File: SDKs/googleapis.com/metastore/v1beta/go/pkg/models/shared/consumer.go
package shared

// Consumer
// Contains information of the customer's network configurations.
type Consumer struct {
	EndpointURI *string `json:"endpointUri,omitempty"`
	Subnetwork  *string `json:"subnetwork,omitempty"`
}

// ConsumerInput
// Contains information of the customer's network configurations.
type ConsumerInput struct {
	Subnetwork *string `json:"subnetwork,omitempty"`
}
